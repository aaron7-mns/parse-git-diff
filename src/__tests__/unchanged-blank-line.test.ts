import { getFixture } from './test-utils';
import parseGitDiff from '../parse-git-diff';

describe('unchanged blank line', () => {
  const fixture = getFixture('unchanged-blank-line');

  it('parse `unchanged-blank-line`', () => {
    expect(parseGitDiff(fixture)).toMatchSnapshot();
  });
});
