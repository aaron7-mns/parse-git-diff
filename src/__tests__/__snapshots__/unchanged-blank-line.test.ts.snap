// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unchanged blank line parse \`unchanged-blank-line\` 1`] = `
{
  "files": [
    {
      "chunks": [
        {
          "changes": [
            {
              "content": "",
              "lineAfter": 4,
              "lineBefore": 4,
              "type": "UnchangedLine",
            },
            {
              "content": "  // unchanged blank line above",
              "lineAfter": 5,
              "lineBefore": 5,
              "type": "UnchangedLine",
            },
            {
              "content": "  console.log("hello world");",
              "lineAfter": 6,
              "type": "AddedLine",
            },
            {
              "content": "",
              "lineAfter": 7,
              "lineBefore": 6,
              "type": "UnchangedLine",
            },
          ],
          "context": "function hello() {",
          "fromFileRange": {
            "lines": 4,
            "start": 4,
          },
          "toFileRange": {
            "lines": 5,
            "start": 4,
          },
          "type": "Chunk",
        },
      ],
      "path": "rename.js",
      "type": "ChangedFile",
    },
  ],
  "type": "GitDiff",
}
`;
